charts/eks-anywhere-packages/values.yaml

```diff
@@ -66,11 +66,13 @@ controller:
   # -- Controller repository name.
   repository: "eks-anywhere-packages"
   # -- Controller image tag, or sha sum.
-  tag: "latest"
+  tag: "v0.1.2-eks-a-v0.0.0-dev-build.1944"
   # -- Additional environment variables for the controller pod.
   env:
     - name: ENABLE_WEBHOOKS
       value: "true"
+    - name: EKSA_PUBLIC_KEY
```

Comment:
This isn't needed as the key is hard coded and the environment variable is only needed to override it.